src/lib/components/Modal/README.mdx

```diff
@@ -825,21 +826,157 @@ To enable it, you just need to pass a reference to the buttons using
 a reference to the button is that if the button is disabled, the key press will
 not fire the event.
 
-👉 We strongly recommend using this feature together with
-[Autofocus](#autofocus) for a better user experience.
-
-## Autofocus
+## Autofocus And Focus Trap
 
 Autofocus is implemented to enhance the user experience by automatically
-focussing an element within the modal.
+focusing an element within the modal.
 
 How does it work? It tries to find `input`, `textarea`, and `select` elements
 inside of Modal and moves focus onto the first non-disabled one. If none is
 found and the `primaryButtonRef` prop on Modal is set, then the primary button
-is focused.
+is focused. If there are neither, it tries to focus any other focusable elements.
+In case there are none, Modal itself is focused.
+
+For most use cases, to make Modal accessible to all users, it should not be possible
+to move focus on elements outside of it.
 
-Autofocus is enabled by default, so if you want to control the focus of
-elements manually, set the `autoFocus` prop on Modal to `false`.
+This behavior is enabled by default and works only if `autoFocus` is enabled.
+We strongly recommend you keep it this way. If you need to move focus outside
+of Modal, set `focusTrap` to `false`.
```

Comment:
So is there a valid case for setting `focusTrap` to `false`?
If not, then why do we support it?

Reply:
> of a chat window or some other widget type elements

I'm not sure if our `Modal` with automatic positioning in the center of the screen and with an opaque backdrop is the correct component. 

I think I would like to hear the arguments supporting this before merge if possible. Could we please postpone merge until out Monday call?

Reply:
Ok, i will add it as a topic on FEK board.

Reply:
Good point @mbohal. While dialogs (and the `<dialog>` element, once we implement it) can be opened in both modal and non-modal way, our use case is limited to the former. Given the component name is `Modal`, we are providing a _modal_ dialog which means the rest of the page is expected to be inert and non-interactive. Then the `focusTrap` option shouldn't be necessary.